src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filterMan = people
+    .filter(man => man.sex === 'm')
+    .filter(man => century ? Math.ceil(man.died / 100) === century : true);
+
+  const ageMan = filterMan.filter(man => man.sex === 'm')
```

Comment:
It seems like you're filtering for men again, which is not necessary because you've already filtered for men on line 10. You can remove this filter.